### PR TITLE
Copy Implementation

### PR DESCRIPTION
**Pull request by emalineg**
1. SEO copy implemented and set to be visible by default.
2. Button at bottom of copy which scrolls users back up to Astrostation and hides copy.
3. Button added to settings which makes copy visible again. 
4. Tooltip added to tutorial for hiding SEO copy.

ONLY ISSUE: 
On mobile, height of Astrostation is not adjusting to flex around inner components, resulting in overlap with InfoSection.

I won't be able to look into this until the weekend, if anyone can pinpoint a fix.

### CODE REVIEW

**Comment by Melkeydev** [comment]
Based on this comment in the description:ONLY ISSUE:
On mobile, height of Astrostation is not adjusting to flex around inner components, resulting in overlap with InfoSection.

I won't be able to look into this until the weekend, if anyone can pinpoint a fix.

We should mark this as WIP because I do not want to merge anything that introduces new known issues

**Comment by emalineg** [comment]
Agreed

File: src/components/Common/Buttons/SeoToggle.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  };
+
+  return(
+<div className="pt-5 pb-5" >
+        <button
+          onClick={handleClick}
+          type="button"
+          className={"seoToggle flex items-center rounded-md bg-violet-700 ml-2 mb-2 px-4 py-2 font-medium text-white shadow-sm focus:outline-none dark:bg-violet-900/[0.6]  dark:text-violet-200"}
+        >
+         Back to Astrostation
```
Comment (royanger) [inline src/components/Common/Buttons/SeoToggle.tsx:19]: Maybe change to `Back to Astrostation (hide info)`? Not sure I love that though.

File: src/pages/Astrostation.tsx [hunk 2]
```diff
@@ -63,7 +66,7 @@ export const HomePage = ({ backgrounds }: { backgrounds: any }) => {
   const { grid } = useGrid();
 
   return (
-    <div className="h-screen">
+    <div ref={ref} className="h-screen">
```
[the remark this answers is not in the document]
Comment (emalineg) [reply src/pages/Astrostation.tsx:69]:
```suggestion
 </div>
      {!isDesktop ? (
        <div className="ml-8 flex flex-col items-center pt-10 pb-40">
```
I tried the md:h-screen, but it caused issues with responsiveness of the widgets. I added pb-40 to this line and it fixed the overlapping, and no other issues are being caused by it. 

Is this an okay fix?

File: src/components/Common/Buttons/SeoToggle.tsx (hunk 1 above)
Comment (emalineg) [reply src/components/Common/Buttons/SeoToggle.tsx:19]: Options:

image
image
image
image
image
image
image
image

File: src/pages/Astrostation.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (royanger) [reply src/pages/Astrostation.tsx:69]: Generally a padding/margin solution for this type of thing isn't a great solution but can work. In this case its not a clean solve. I'm seeing the following:

[the remark this answers is not in the document]
Comment (royanger) [reply src/pages/Astrostation.tsx:69]: What's the responsive issue you were seeing? I gave line 69 the following classes and it seems fine with a quick look. `md:h-screen pb-8 md:pb-0` See the following image.

File: src/components/Common/Buttons/SeoToggle.tsx (hunk 1 above)
Comment (royanger) [reply src/components/Common/Buttons/SeoToggle.tsx:19]: Any of the options that have some form of hide info/hide section look fine to me. I just felt being a little explicit was a good thing.

File: src/pages/Astrostation.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (emalineg) [reply src/pages/Astrostation.tsx:69]: The issue was when you manually dragged a window to become smaller and then back to larger. The widgets would all pile on top of each other top and center of the page.

**Pull request merged**